testhelpers/consensus.go

```diff
@@ -185,15 +185,21 @@ func newMessageApplier(smsg *types.SignedMessage, processor *consensus.DefaultPr
 	return nil, err
 }
 
-// CreateAndApplyTestMessage wraps the given parameters in a message and calls ApplyTestMessage
-func CreateAndApplyTestMessage(t *testing.T, st state.Tree, vms vm.StorageMap, to address.Address, val, bh uint64, method string, ancestors []types.TipSet, params ...interface{}) (*consensus.ApplicationResult, error) {
+// CreateAndApplyTestMessageFrom wraps the given parameters in a message and calls ApplyTestMessage.
```

Comment:
With new storagemarket tests for the first time the `From` address of a message has become semantically important so this refactor allows tests to set this field.